src/integrationTest/sam.test.ts

```diff
@@ -233,7 +233,12 @@ async function getAddConfigCodeLens(documentUri: vscode.Uri): Promise<vscode.Cod
     return waitUntil(
         async () => {
             try {
-                let codeLenses = await getCodeLenses(documentUri)
+                // Sometimes 'getCodeLenses' will never return, so we need to race it
+                let codeLenses = await Promise.race<Promise<vscode.CodeLens[] | undefined>>([
```

Comment:
Does this change make the old CODELENS_RETRY_INTERVAL usage (now line 264) redundant?

Reply:
Good question. The 2nd retry interval is used for when `getCodeLens` throws an error before the other Promise resolves. The code would actually now wait a total of 2*`CODELENS_RETRY_INTERVAL` if `getCodeLens` stalls.